docs/data-management-on-carbondata.md

```diff
@@ -730,6 +736,8 @@ Users can specify which columns to include and exclude for local dictionary gene
   * If the IGNORE option is used, then bad records are neither loaded nor written to the separate CSV file.
   * In loaded data, if all records are bad records, the BAD_RECORDS_ACTION is invalid and the load operation fails.
   * The maximum number of characters per column is 32000. If there are more than 32000 characters in a column, data loading will fail.
+  * Since Bad Records Path can be specified in both create, load and carbon properties. 
```

Comment:
"both" does not suite in this statement. Please rewrite.

Reply:
done